mods/persistence/modules/mob/new_player.dm

```diff
@@ -186,10 +186,9 @@
 	if(GAME_STATE < RUNLEVEL_GAME)
 		to_chat(src, SPAN_NOTICE("Wait until the round starts to join."))
 		return
-	if(!config.enter_allowed)
```

Comment:
You probably need an additional check on line 137 in the ``characterSelect`` proc.

Reply:
trueee